### PR TITLE
test: Update ClusterFuzzLite github->action

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Pull Request

## Description

* Update github->action
* Add first test fuzzer file

## Type of change

- [x] CI system update
- [x] Test Coverage update

## Testing
Every time there is a push to the main branch, fuzzing analysis will be launched

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] My code meets the required code coverage for lines (90% and above)
- [ ] My code meets the required code coverage for branches (80% and above)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        // Fuzzing input data
+        webhookConfig.setRepositoryUrl(data.consumeRemainingAsString());
+        webhookConfig.setPullRequestUrl(data.consumeRemainingAsString());
+        
+        // Perform the fuzz test
+        try {
+            String result = LPVSWebhookUtil.getPullRequestId(webhookConfig);
+
+            // Validate the result
+            // assertNotNull(result);
```
Comment (o-kopysov) [inline src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java:28]: Why do you comment this check?

Comment (tdrozdovsky) [reply src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java]: Reverted

**Review by o-kopysov — APPROVED** [review]
LGTM

File: src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package com.lpvs.util;
```
Comment (MoonkiHong) [inline src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java:1]: We need the copyright claimer as well as the license text here.

**Review by MoonkiHong — CHANGES_REQUESTED** [review]
PTAL.

File: src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java (hunk 2 above)
Comment (tdrozdovsky) [reply src/test/java/com/lpvs/util/LPVSWebhookUtilTestFuzzer.java:7]: Done!

**Review by MoonkiHong — APPROVED** [review]
+1

**Pull request merged**